src/Components/test/testassets/BasicTestApp/BasicTestApp.csproj

```diff
@@ -7,6 +7,9 @@
     <ReferenceBlazorBuildLocally>true</ReferenceBlazorBuildLocally>
     <!-- Must be defined before ReferenceFromSource.props is imported -->
     <AdditionalRunArguments>--pathbase /subdir</AdditionalRunArguments>
+
+    <!-- Resx generation is custom, preserve current logic -->
+    <GenerateResxSource>false</GenerateResxSource>
```

Comment:
I'm assuming the logic is custom due to test specific requirements.

Reply:
From offline discussion, it seems like we want to generate the .cs file for the non localized resource but don't generate it for the localized version? Is that correct?